### PR TITLE
Chat Rebuilt with Pusher

### PR DESCRIPTION
**Pull request by jazwu**
## Description
- rewrite frontend and backend with pusher
- set pusher key in settings.py
- reset server back to wsgi

## How did you test the changes?
image
image

### CODE REVIEW

File: .gitignore
Comment (shreyas2499) [inline .gitignore:1]: Please remove this file. We wouldn't need this

File: healthConfig/settings.py [hunk 1]
```diff
@@ -164,3 +156,9 @@
 AWS_DEFAULT_ACL = None
 AWS_S3_VERITY = True
 DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+
+# Pusher Configuration
+PUSHER_APP_ID = "1793983"
```
Comment (shreyas2499) [inline healthConfig/settings.py]: We can add these keys in the environment variables

File: requirements.txt [hunk 2]
```diff
@@ -33,20 +33,25 @@ incremental==22.10.0
 jmespath==1.0.1
 mccabe==0.7.0
 mypy-extensions==1.0.0
+mysql-connector==2.2.9
```
Comment (shreyas2499) [inline requirements.txt]: Please remove these two mysql packages. They cause issues with deployment

File: healthConfig/settings.py (hunk 1 above)
Comment (jazwu) [reply healthConfig/settings.py]: should we set this on aws?

Comment (shreyas2499) [reply healthConfig/settings.py]: We'll set on both aws and travis ci

Comment (jazwu) [reply healthConfig/settings.py]: ok! you and Sarthak will set them for us right? I'll just leave keys here

Comment (shreyas2499) [reply healthConfig/settings.py]: PUSHER_APP_ID
PUSHER_KEY	
PUSHER_SECRET

These are the key names that I have sent on travis and aws. We can use these

Comment (jazwu) [reply healthConfig/settings.py]: modified

File: requirements.txt [hunk 3]
```diff
@@ -33,20 +33,24 @@ incremental==22.10.0
 jmespath==1.0.1
 mccabe==0.7.0
 mypy-extensions==1.0.0
+mysql-connector-python==8.3.0
```
Comment (shreyas2499) [inline requirements.txt:36]: Please remove this package as well!

**Review by shreyas2499 — APPROVED** [review]

**Pull request merged**